src/api/app/models/user.rb

```diff
@@ -66,10 +66,12 @@ class User < ApplicationRecord
   has_and_belongs_to_many :announcements
   has_many :commit_activities
 
+  scope :confirmed, -> { where(state: 'confirmed') }
   scope :all_without_nobody, -> { where('login != ?', NOBODY_LOGIN) }
   scope :not_deleted, -> { where.not(state: 'deleted') }
   scope :not_locked, -> { where.not(state: 'locked') }
   scope :with_login_prefix, ->(prefix) { where('login LIKE ?', "#{prefix}%") }
+  scope :active, -> { confirmed.or(User.where(state: :subaccount, owner: User.confirmed)) }
```

Comment:
Wasn't there a problem with using scopes within scopes a couple of weeks ago?

Reply:
Jein - the problem was that calling functions from within a scope uses that scope as default scope. Not problematic here